### ISSUE TITLE
more accessible font weight/size choices

### ISSUE THREAD

**Issue by kenirwin** [alaoweb/2021.alaoweb.org#5]
Most of the text on our front page, and on other page headers & footers falls below WCAG contrast ratios, both due to lack of contrast with the background and due to small font sizes.
